<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PurchaseReport</class>
 <widget class="QDialog" name="PurchaseReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 254, 178);
</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>931</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font-size:30px;
color:green</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Покупка&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="purchaseTableWidget">
      <property name="styleSheet">
       <string notr="true">color: black;
background-color: lightgreen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="logoutButton">
      <property name="styleSheet">
       <string notr="true">color: black;
background-color: lightgreen</string>
      </property>
      <property name="text">
       <string>Закрыть</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>516</x>
     <y>0</y>
     <width>441</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Поиск по названию номенклатуры</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditSearch">
      <property name="styleSheet">
       <string notr="true">border: 1px solid black; background-color:white;
</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
